exotica/src/KinematicTree.cpp

```diff
@@ -907,20 +907,21 @@ void KinematicTree::resetJointLimits()
 
     ///	Manually defined floating base limits
     ///	Should be done more systematically with robot model class
+    const double inf = std::numeric_limits<double>::infinity();
```

Comment:
Can you use a `constexpr` here? And also use a definition for pi: `constexpr double pi() { return std::atan(1)*4; }`

Reply:
Infinity may actually not play nicely with sampling-based planners from OMPL, I realised.

Reply:
> Infinity may actually not play nicely with sampling-based planners from OMPL, I realised.

Right :-) But does it recognise user-chosen joint limits? In this case, a user has to select translation limits before sampling from bounds.

Reply:
We have exposed ``setFloatingBaseLimits`` so that could work _if_ a user remembers to do so. I'll have a look at this PR later again when CI passed (but that may require some formatting).

Reply:
> constexpr double pi() { return std::atan(1)*4; }

Why not use `M_PI`? I understand that `constexpr` is an inline hint. Since `M_PI` is a macro, it will be inline and not changeable.

Reply:
Cf. e.g. [here](https://meetingcpp.com/blog/items/cpp-and-pi.html)

Reply:
Alright. Do we have a place in exotica (maybe a "math_defines" header?) which collects all these constants?

Reply:
I placed both as `constexpr` just in the function.